api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -300,6 +364,17 @@ message RedisProxy {
   // client. If an AUTH command is received when the password is not set, then an "ERR Client sent
   // AUTH, but no ACL is set" error will be returned.
   config.core.v3.DataSource downstream_auth_username = 7 [(udpa.annotations.sensitive) = true];
+
+  // Enable some extra features.
+  //
+  // Example:
+  //
+  // .. code-block:: yaml
+  //
+  //    feature_config:
+  //      hotkey: {}
+  //
+  FeatureConfig feature_config = 9;
```

Comment:
From an API perspective I think I would just go ahead and inline `HotKey` here.

Reply:
Yes, you are right. But I want to enhance future extensions. Some extra features are all in FeatureConfig, such as HotKey or other.

Reply:
If we have future features we can add them directly. I don't think this later of indirection adds any value.